docs/docs/components/api/README.md

```diff
@@ -139,6 +139,7 @@ props: {
 | `label`          | `string`                             | optional  | A friendly label to show to user for this prop. If a label is not provided, the `propName` is displayed to the user.                                                                                                                                                                                                                                                                                                                                                           |
 | `description`    | `string`                             | optional  | Displayed near the prop input. Typically used to contextualize the prop or provide instructions to help users input the correct value. Markdown is supported.                                                                                                                                                                                                                                                                                                                  |
 | `options`        | `string[]` or `object[]` or `method` | optional  | Provide an array to display options to a user in a drop down menu.<br>&nbsp;<br>**`[]` Basic usage**<br>Array of strings. E.g.,<br>`['option 1', 'option 2']`<br>&nbsp;<br>**`object[]` Define Label and Value**<br>`[{ label: 'Label 1', value: 'label1'}, { label: 'Label 2', value: 'label2'}]`<br>&nbsp;<br>**`method` Dynamic Options**<br>You can generate options dynamically (e.g., based on real-time API requests with pagination). See configuration details below. |
+| `useQuery`    | `boolean`                             | optional  | Use in conjunction with **Dynamic Options**. If set to `true`, the prop will accept a real-time query that can be used by the `options` method to obtain results according to that query.                                                                                                                                                                                                                                                                                                                   |
```

Comment:
```suggestion
| `useQuery`    | `boolean`                             | optional  | Use in conjunction with **Dynamic Options**. If set to `true`, the prop accepts a real-time query that can be used by the `options` method to obtain results according to that query.                                                                                                                                                                                                                                                                                                                   |
```

Reply:
Changed 'will accept' to 'accepts' for consistency with Google's Developer docs style guide (use present tense).